tests/conftest.py

```diff
@@ -268,6 +265,36 @@ def rl_pipeline_config(embedding_dims):
     return config
 
 
+@pytest.fixture()
+def rl_pipeline_v1_config(_base_rl_pipeline_config):
+    return {
+        **{
+            K: 3,
+            SYNTHETIC_PARAMS: {
+                K: 3,
+                INTERACTIONS_RANGE: (1, 2),
+                REWARD_GENERATION_MODE: RewardGeneration.COMPLEX,
+            },
+        },
+        **_base_rl_pipeline_config,
+    }
+
+
+@pytest.fixture()
+def rl_pipeline_v2_config(_base_rl_pipeline_config):
+    return {
+        **{
+            K: 2,
+            SYNTHETIC_PARAMS: {
+                K: 2,
+                INTERACTIONS_RANGE: (1, 2),
+                REWARD_GENERATION_MODE: RewardGeneration.COMPLEX,
+            },
+        },
+        **_base_rl_pipeline_config,
+    }
```

Comment:
On the MP side, we removed the v2 version entirely. I believe we may safely erase RL v2. But this may be an issue for the future?